### DISCUSSION TITLE
Submit on https://webcatalog.io

### DISCUSSION THREAD

**Discussion by jayantikatariya** [ankurk91/google-chat-electron#107] [Ideas]
It will be great if you submit on https://webcatalog.io so easy to install and update for all.

**Comment by ankurk91** [comment]
we already support all of 3 desktop platform, why is the benefit of submitting it to webcatalog?

**Comment by jayantikatariya** [comment]
Easy to install and update from there. :-)

**Comment by ankurk91** [comment]
![Screenshot from 2022-08-08 11-38-14](https://user-images.githubusercontent.com/6111524/183350684-0de2b154-7fe7-4e98-9b2e-da19c4f1da52.png)

Their platform is broken (see screenshot).
Anyone can submit the app to their catalog, feel free to submit.
